iopmp_ref_model/include/config.h

```diff
@@ -1,4 +1,8 @@
 /***************************************************************************
+// Copyright (c) 2025 by 10xEngineers.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
```

Comment:
You mention `LICENSE` here but there is no such file in this PR.
We have Apache 2.0 `LICENSE` under `libiopmp` folder: https://github.com/riscv-non-isa/iopmp-spec/blob/main/libiopmp/LICENSE.
Just copy that file to `iopmp_ref_model` folder, I think.